### PR TITLE
[Sw-109, SW-110, SW-24] Account details popup

### PR DESCRIPTION
**Pull request by ifaouibadi**

### CODE REVIEW

**Review by CedrikNikita — COMMENTED** [review]
Is this behaviour is intendent @paolomolo?

Account popup behaviour

File: src/popup/router/components/Modals/AccountDetails.vue [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+            @click.prevent="activeTab = 'transactions'"
+          >
+            {{ $t('pages.token-details.transactions') }}
+          </ButtonPlain>
+          <ButtonPlain
+            :class="{active: activeTab === 'names'}"
+            @click.prevent="activeTab = 'names'"
+          >
+            {{ $t('pages.titles.names') }}
+          </ButtonPlain>
```
Comment (pstachel) [inline src/popup/router/components/Modals/AccountDetails.vue:52]: To not repeat the code I'd suggest to make an array with all the tab names
```
tabs: ['tokens', 'transactions', 'names'],
```

and then iterate through them with v-for

```
<ButtonPlain
            v-for="tab in tabs"
            :key="tab"
            :class="{ active: activeTab === tab }"
            @click.prevent="setActiveTab(tab)"
          >
            {{ $t(`pages.account-details.${tab}') }}
          </ButtonPlain>
```

Of course translation strings should be adjusted accordingly

File: src/popup/router/components/Modals/AccountDetails.vue [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+
+.account-details {
+  font-weight: 500;
+  color: variables.$color-white;
+
+  ::v-deep .container {
+    padding: 0;
+    width: 360px;
+    height: 600px;
+    overflow: hidden;
```
Comment (pstachel) [inline src/popup/router/components/Modals/AccountDetails.vue:166]:
```suggestion
    overflow: hidden auto;
```

or 
```
overflow-x: hidden;
overflow-y: auto;
```

File: src/popup/router/components/Modals/AccountDetails.vue [hunk 3]
```diff
@@ -0,0 +1,332 @@
...
+      width: 75px;
+      height: 58px;
+      background-color: rgba(variables.$color-white, 0.08);
+      border-radius: 10px;
+      color: rgba(variables.$color-white, 0.75);
+      text-decoration: none;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
```
Comment (pstachel) [inline src/popup/router/components/Modals/AccountDetails.vue:237]: You could use flex mixin

File: src/popup/router/components/TransactionList.vue [hunk 4]
```diff
@@ -66,7 +66,8 @@ export default {
       isDestroyed: false,
       displayMode: { rotated: true, filter: 'all', sort: 'date' },
       filters: {
-        all: {}, sent: {}, received: {}, tips: {}, date: { rotated: true },
+        all: {}, in: {}, out: {}, dex: {},
+        // date: { rotated: true },
```
Comment (pstachel) [inline src/popup/router/components/TransactionList.vue:70]: do we still need it?

**Review by pstachel — CHANGES_REQUESTED** [review]

File: src/popup/router/components/Modals/AccountDetails.vue [hunk 5]
```diff
@@ -0,0 +1,348 @@
...
+
+      if (this.activeTab === 'transactions') {
+        return this.$t('pages.recentTransactions.searchPlaceholder');
+      }
+
+      return null;
+    },
+  },
+  mounted() {
+    if (this.$refs.account_details) {
```
Comment (CedrikNikita) [inline src/popup/router/components/Modals/AccountDetails.vue:133]: We have an agreement to name variables in camelCase.

**Comment by CedrikNikita** [comment]
You need to fix all the lgtm-bot problems, plus first commit looks weird.

File: src/popup/router/components/AccountCard.vue [hunk 6]
```diff
@@ -2,11 +2,17 @@
...
-    <AccountInfo
-      v-bind="$attrs"
-      :color="color"
-    />
+    <div @click.stop="">
+      <AccountInfo
+        v-bind="$attrs"
+        :color="color"
+      />
+    </div>
```
Comment (CedrikNikita) [inline src/popup/router/components/AccountCard.vue:15]:
```suggestion
    <AccountInfo
      v-bind="$attrs"
      :color="color"
      @click.native.stop
    />
```

File: src/popup/router/components/AccountCard.vue [hunk 7]
```diff
@@ -17,7 +23,10 @@
           {{ $t('pages.fungible-tokens.tokens') }}
         </span>
       </div>
-      <div class="buttons">
+      <div
+        class="buttons"
+        @click.stop=""
```
Comment (CedrikNikita) [inline src/popup/router/components/AccountCard.vue:28]:
```suggestion
        @click.stop
```

**Pull request merged**